backend/src/migrations/20230828113106_add_hard_skills_table.ts

```diff
@@ -0,0 +1,39 @@
+import { type Knex } from 'knex';
+
+const uuid = 'uuid_generate_v4()';
+const constraintName = 'hard_skills_pkey';
+
+const TABLE_NAME = 'hard_skills';
+
+const ColumnName = {
+    ID: 'id',
+    NAME: 'name',
+    CREATED_AT: 'created_at',
+    UPDATED_AT: 'updated_at',
+};
+
+function up(knex: Knex): Promise<void> {
+    return knex.schema.createTable(TABLE_NAME, (table) => {
+        table
```

Comment:
When you use uuid function first of all you need to create extension if not exists, because in other db this migration can fall down without the necessary extension

```suggestion
function up(knex: Knex): Promise<void> {
    await knex.raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";');
    
    return knex.schema.createTable(TABLE_NAME, (table) => {
        table
```